### PR TITLE
Compression

### PR DESCRIPTION
**Pull request by dcbroad3**
Adds gzip compression to all SDS WaveForm samples. Also adds description to Readme files. Python requests module automatically handles compression internally, so also added a note to the SDS TimeSeries Python sample to indicate this.

### CODE REVIEW

File: basic_samples/SDS/README.md [hunk 1]
```diff
@@ -1,48 +1,42 @@
...
....visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSDotNetAPI?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4914&branchName=master) <br /> [![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSJava?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4910&branchName=master) <br />[![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSangJS?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4923&branchName=master)<br />[![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSnodeJS?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4922&branchName=master) <br /> [![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSPy?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4907&branchName=master) |
```
Comment (derekendres) [inline basic_samples/SDS/README.md]:
```suggestion
| <a href="DotNet/SdsClientLibraries/SdsClientLibraries">.NET</a><br /><a href="DotNet/SdsRestApiCore/SdsRestApiCore">.NET\*</a><br /><a href="Java/sdsjava">Java</a><br /><a href="JavaScript/Angular">Angular</a><br /><a href="JavaScript/NodeJs">nodeJS</a><br /><a href="Python/SDSPy/Python3">Python3</a> | [![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSDotNet?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4912&branchName=master) <br />[![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSDotNetAPI?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4914&branchName=master) <br /> [![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSJava?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4910&branchName=master) <br />[![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSangJS?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4923&branchName=master)<br />[![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSnodeJS?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4922&branchName=master) <br /> [![Build Status](https://osisoft.visualstudio.com/Engineering%20Incubation/_apis/build/status/All_Test/SDSPy?branchName=master)](https://osisoft.visualstudio.com/Engineering%20Incubation/_build/latest?definitionId=4907&branchName=master) |
```

**Comment by derekendres** [comment]
Is there compression shown for Python?

**Comment by dcbroad3** [comment]
> 
> 
> Is there compression shown for Python?

It's mentioned here: https://github.com/osisoft/OSI-Samples-OCS/tree/compression/basic_samples/SDS/Python/SDSPy/Python3#establish-a-connection
Basically, the 'requests' Python module that we do automatically does compression under the hood by default, there's no action required.

**Comment by derekendres** [comment]
I swear I read that area like 5times and somehow missed it... 👍

**Pull request merged**